commit 79bebadd4e0e6d71c5c889743b3b42c6e788f8a6
Author: Ashok Raj <ashok.raj@intel.com>
Date:   Mon Jan 30 09:39:53 2017 -0800

    iommu/vt-d: Tylersburg isoch identity map check is done too late.
    
    commit 21e722c4c8377b5bc82ad058fed12165af739c1b upstream.
    
    The check to set identity map for tylersburg is done too late. It needs
    to be done before the check for identity_map domain is done.
    
    To: Joerg Roedel <joro@8bytes.org>
    To: David Woodhouse <dwmw2@infradead.org>
    Cc: iommu@lists.linux-foundation.org
    Cc: linux-kernel@vger.kernel.org
    Cc: Ashok Raj <ashok.raj@intel.com>
    
    Fixes: 86080ccc22 ("iommu/vt-d: Allocate si_domain in init_dmars()")
    Signed-off-by: Ashok Raj <ashok.raj@intel.com>
    Reported-by: Yunhong Jiang <yunhong.jiang@intel.com>
    Signed-off-by: Joerg Roedel <jroedel@suse.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1248_79bebadd4e0e6d71c5c889743b3b42c6e788f8a6.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1248_79bebadd4e0e6d71c5c889743b3b42c6e788f8a6.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
